packages/create-invoice-form/src/lib/utils/prepareRequest.ts

```diff
@@ -3,116 +3,122 @@ import type { CustomFormData } from "@requestnetwork/shared";
 import { parseUnits, zeroAddress } from "viem";
 
 interface IRequestParams {
-  currency: string;
-  formData: CustomFormData;
-  invoiceTotals: {
-    amountWithoutTax: number;
-    totalTaxAmount: number;
-    totalAmount: number;
-  };
-  signer: string;
-  currencies: any;
+	currency: string;
```

Comment:
Why switch to 4 spaces instead of 2? Seems like noise to me.

Reply:
Oh i'm sorry might be my VSCode settings messed up.